parquet-common/src/main/java/org/apache/parquet/conf/ParquetConfiguration.java

```diff
@@ -0,0 +1,178 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.parquet.conf;
+
+import java.util.Map;
+
+/**
+ * Configuration interface with the methods necessary to configure Parquet applications.
+ */
+public interface ParquetConfiguration extends Iterable<Map.Entry<String,String>> {
+
+  /**
+   * Sets the value of the name property.
+   *
+   * @param name the property to set
+   * @param value the value to set the property to
+   */
+  void set(String name, String value);
```

Comment:
setString()?

Reply:
I just went with the way Hadoop's `Configuration` does it for this and other methods in the interface, in order to not have to change references to conf.set() everywhere.